### DISCUSSION TITLE
Key Binding Names For Custom Scenes?

### DISCUSSION THREAD

**Discussion by BeatFitnesse** [kinsi55/CS_BeatSaber_Camera2#103] [Q&A] [answered]
How does one know what a particular key is named when setting up bindings for custom scenes?  For example, if I want Scene 1 bound to the trigger on my left controller, and scene 2 bound to the trigger on my right, how do I know what name to enter into the Json for scenes config?  SteamVR seems to just call each one "Trigger."  How do I know what the mod will pick up?

**Comment by kinsi55** [comment, marked as answer]
The Key bindings in this case refer to Keyboard keys - You cannot bind them to your VR inputs
